application/src/main/java/run/halo/app/content/comment/ReplyService.java

```diff
@@ -31,7 +31,7 @@ static Comparator<Reply> creationTimeAscComparator() {
             reply -> reply.getMetadata().getCreationTimestamp();
         // ascending order by creation time
         // asc nulls high will be placed at the end
-        return Comparator.comparing(creationTime, Comparators.nullsLow())
+        return Comparator.comparing(creationTime, Comparators.nullsHigh())
```

Comment:
See https://github.com/spring-projects/spring-framework/commit/a01384068ab0c3124e8ca9a48b0fe5de873d5097 for more.